cypress/integration/gift_cards.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {HomePo} from "../support/home.po";
+import {GiftCardsPo} from "../support/giftCards.po";
+
+describe('Gift cards suite', () => {
+    const homePage = new HomePo();
+    const giftCardPage = new GiftCardsPo();
+
+    beforeEach(() => {
+
+        cy.visit('');
+    })
+
+    it('should ', () => {
+        const price = '100';
+        const giftCardId = '[href*="/B07TMMZKBJ"]';
+
+        homePage.clickOnElem(homePage.giftCardLink);
+        giftCardPage.isCurrentURLContain(giftCardPage.pageUrl);
+        for (let elem of giftCardPage.giftCardLinks) {
+            giftCardPage.isElemVisible(elem);
+        }
+        giftCardPage.clickOnElem(giftCardPage.giftCardLinks[0]);
+        giftCardPage.isElemVisible(giftCardPage.customGiftAmountInput);
+        giftCardPage.clickOnElem(giftCardPage.customGiftAmountField);
+        giftCardPage.typeIntoElement(giftCardPage.customGiftAmountInput, price);
+        giftCardPage.isElemContainText(giftCardPage.giftCardAmount, price);
+        giftCardPage.typeIntoElement(giftCardPage.emailField, 'email@email.com');
```

Comment:
You've checked manual setting gift amount. This is a good way for checking that functionality. But in our task was other case:  When user set 100$ as gift ammount - Then this item become selected and price changed to 100$
Maybe, you were confused, that $100 is not visible in your resolution. If so - you can set         cy.viewport(1400, 800);